Microsoft.Toolkit.Uwp.UI.Controls/Graph/SharePointFiles/SharePointFiles.Properties.cs

```diff
@@ -0,0 +1,245 @@
+            DependencyProperty.Register(
+                nameof(FileSize),
+                typeof(long),
+                typeof(SharePointFiles),
+                new PropertyMetadata(0L));
+
+        internal static readonly DependencyProperty LastModifiedProperty =
+            DependencyProperty.Register(
+                nameof(LastModified),
+                typeof(string),
+                typeof(SharePointFiles),
+                null);
+
+        private static readonly DependencyProperty IsDetailPaneVisibleProperty =
+            DependencyProperty.Register(
+                nameof(IsDetailPaneVisible),
+                typeof(bool),
+                typeof(SharePointFiles),
+                new PropertyMetadata(false));
+
+        private int _fileUploading;
+        private string _errorMessage;
+
+        /// <summary>
+        /// Gets or sets token to access Microsoft Graph API
+        /// </summary>
+        public string GraphAccessToken
+        {
+            get { return ((string)GetValue(GraphAccessTokenProperty))?.Trim(); }
+            set { SetValue(GraphAccessTokenProperty, value?.Trim()); }
+        }
+
+        /// <summary>
+        /// Gets or sets url of OneDrive to display
+        /// </summary>
+        public string DriveUrl
+        {
+            get { return ((string)GetValue(DriveUrlProperty))?.Trim(); }
+            set { SetValue(DriveUrlProperty, value?.Trim()); }
+        }
+
+        /// <summary>
+        /// Gets or sets how DetailPane shows
+        /// </summary>
+        public DetailPaneDisplayMode DetailPane
+        {
+            get { return (DetailPaneDisplayMode)GetValue(DetailPaneProperty); }
+            set { SetValue(DetailPaneProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets page size of each request
+        /// </summary>
+        public int PageSize
+        {
+            get { return (int)GetValue(PageSizeProperty); }
+            set { SetValue(PageSizeProperty, value); }
+        }
+
+        internal bool HasMore
+        {
+            get { return (bool)GetValue(HasMoreProperty); }
+            set { SetValue(HasMoreProperty, value); }
+        }
+
+        internal DriveItem SelectedFile
+        {
+            get { return (DriveItem)GetValue(SelectedFileProperty); }
+            set { SetValue(SelectedFileProperty, value); }
+        }
+
+        internal long FileSize
+        {
+            get { return (long)GetValue(SizeProperty); }
+            set { SetValue(SizeProperty, value); }
+        }
+
+        internal string LastModified
+        {
+            get { return (string)GetValue(LastModifiedProperty); }
+            set { SetValue(LastModifiedProperty, value); }
+        }
+
+        private int FileUploading
+        {
+            get
+            {
+                return _fileUploading;
+            }
+
+            set
+            {
+                _fileUploading = value;
+                if (value > 0)
```

Comment:
You shouldn't be setting all these UI properties in the setter of a property.  This should all be pushed to XAML.